addons/network/XEH_preInit.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+#include "initSettings.sqf"
+
+// Restore loadouts lost by the naked unit bug
+[QGVAR(validateLoadout), {
+    params ["_unit", "_loadout"];
+
+    if (GVAR(LoadoutValidation) > 0 && !(getUnitLoadout _unit isEqualTo _loadout)) then {
+        _unit setUnitLoadout _loadout;
+    };
+}] call CBA_fnc_addEventHandler;
+
+["CAManBase", "Local", {
+    params ["_unit", "_local"];
+    if !(_local) then {
+        // Broadcast loadout to new owner if unit was once local on this machine
+        if (GVAR(LoadoutValidation) > 0 && {_unit getVariable [QGVAR(wasLocal), false]}) then {
+            if (GVAR(LoadoutValidation) == 1 && {playableUnits findIf {_x == _unit} == -1}) exitWith {};
```

Comment:
```sqf
             if (GVAR(LoadoutValidation) == 1 && {playableUnits findIf {_x == _unit} == -1}) exitWith {};
```
Why?????????????????????

```suggestion
            // Exit if unit is not playable and that option is enabled.
            if (GVAR(LoadoutValidation) == 1 && {!(_unit in playableUnits)}) exitWith {};
```

Reply:
I second that "why?". No idea what I was thinking back then😅